docs/docs/widgets/image.md

```diff
@@ -6,21 +6,21 @@ title: Image
 
 The **Image** widget is used to display images in your app.
 
-<div style={{paddingTop:'24px', paddingBottom:'24px'}}>
+<div style={{paddingTop:'24px'}}>
 
 ## Properties
 
 |  <div style={{ width:"100px"}}> Properties </div> |  <div style={{ width:"100px"}}> Description </div> | 
 |:----------- |:----------- |
 | URL | Enter the URL of the image to display it on the widget. |
-| Loading state | Loading state can be used to show a spinner as the image content. Loading state is commonly used with `isLoading` property of the queries to show a loading status while a query is being run. Switch the toggle **On** or click on `fx` to programmatically set the value `{{true}}` or `{{false}}`. |
+| Loading state | Loading state can be used to show a spinner as the image content. Loading state is commonly used with **isLoading** property of the queries to show a loading status while a query is being run. Switch the toggle **On** or click on **fx** to programmatically set the value **{{true}}** or **{{false}}**. |
```

Comment:
Hey, here only fx was needed to be in bold.
Can you please revert the changes you made on isLoading, true and false?
Please update this in the whole doc and both versions.